mmcv/ops/saconv.py

```diff
@@ -120,7 +124,8 @@ def forward(self, x):
             if TORCH_VERSION < '1.5.0' or TORCH_VERSION == 'parrots':
                 out_l = super().conv2d_forward(x, weight)
             else:
-                out_l = super()._conv_forward(x, weight)
+                # bias is a required argument of _conv_forward in torch 1.9.0
```

Comment:
Bias is also required by torch1.8. You can check the version of torch to handle this